SwrveSDK/src/main/java/com/swrve/sdk/ISwrveBase.java

```diff
@@ -110,4 +113,12 @@
     Context getContext();
 
     C getConfig();
+
+    List<SwrveBaseCampaign> getCampaigns();
+
+    List<SwrveBaseCampaign> getCampaigns(SwrveOrientation orientation);
```

Comment:
getCampaigns is the wrong name for this api, so I don't think we should use that.